.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
```

Comment:
Add a limit like snapshot repo if you want to maintain these actively, else we get a lot of PRs and changes.

Reply:
What limit ? This file has been copy/paste from snapshot repo